net-misc/dropbear/dropbear-2018.76.ebuild

```diff
@@ -1,9 +1,9 @@
-# Copyright 1999-2018 Gentoo Foundation
+# Copyright 1999-2018 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI="5"
+EAPI="7"
```

Comment:
This change warrants a revision bump, 
https://devmanual.gentoo.org/general-concepts/ebuild-revisions/index.html